### PR TITLE
Schema registry migrate k8s /ci tests to testcontainer setup

### PR DESCRIPTION
**Pull request by nattilabalint**

### CODE REVIEW

File: testcontainers/build.gradle [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    testImplementation(project(':atlas-integration:atlas-module')) { transitive = false }
+
+    testImplementation(project(':storage:storage-tool')) { transitive = false }
+    testImplementation(project(':storage:storage-core')) { transitive = false }
+    testImplementation(project(':storage:storage-common')) { transitive = false }
+
+    testImplementation libraries.lombok
+    testAnnotationProcessor libraries.lombok
+
+    testImplementation('com.cloudera.dim.kafka-connect:tests:0.0.1.7.2.17.0-SNAPSHOT')
```
Comment (gkomlossi) [inline testcontainers/build.gradle:46]: This version is not reachable, it breaks the compilation.

Comment (gkomlossi) [reply testcontainers/build.gradle:46]: More precisely, the whole library is unavailable

Comment (nattilabalint) [reply testcontainers/build.gradle]: Thanks, fixed.

**Review by gkomlossi — APPROVED** [review]
LGTM

**Review by gkomlossi — APPROVED** [review]

**Pull request merged**